### PR TITLE
fontawesome:0.5.0

### PR DESCRIPTION
**Pull request by duskmoon314**
I am submitting
- [ ] a new package
- [x] an update for a package

In type 0.12.0, a warning will be thrown if a font is not found. This causes several warnings when using the pro sets. The new version adds a state to control the pro sets. If the state is set to true, the pro sets will be used; otherwise, the pro sets will not be used.

Users will not receive unexpected warnings in the new version.

### CODE REVIEW

**Comment by duskmoon314** [comment]
Where can I see the rules of code styles?

**Comment by duskmoon314** [comment]
Oh, I found it in README:

> Style: No specific code style is mandated, but two spaces of indent and kebab-case for variable and function names are recommended.

File: packages/preview/fontawesome/0.5.0/README.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+To use other sets or specify one set, you can pass the `font` parameter to the inner `text` function: \
+`fa-icon("github", font: "Font Awesome 6 Pro Solid")`
+
+If you have Font Awesome Pro, please help me test the library with the Pro set.
+Any feedback is appreciated.
+
+##### Enable Pro sets
+
+Typst 0.12.0 raise a warning when the font is not found.
+To use the Pro set, `typst #fa_use_pro()` should be called before any `fa-*` functions.
```
Comment (elegaanz) [inline packages/preview/fontawesome/0.5.0/README.md:98]: If you want to have Typst syntax highlighting, I think the correct syntax is with three backticks. Also, the function should be renamed here too.

Comment (duskmoon314) [reply packages/preview/fontawesome/0.5.0/README.md:98]: Oh, this is a copy-paste mistake.

Thanks for helping to find the remaining unchanged function names. A commit will be pushed ASAP

**Comment by elegaanz** [comment]
Thanks :)

**Pull request merged**